models/token.go

```diff
@@ -69,6 +85,12 @@ func GetAccessTokenBySHA(sha string) (*AccessToken, error) {
 	return t, nil
 }
 
+// ListAccessTokensByName returns a list of access tokens belongs to given user with name.
+func ListAccessTokensByName(uid int64, name string) ([]*AccessToken, error) {
```

Comment:
What is the use case of this function?

Reply:
probably none as it will in the future only return 0 or 1